pkgs/development/libraries/boxfort/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, coreutils, fetchFromGitHub, cmake, meson, ninja, pkg-config, python3, git }:
+
+stdenv.mkDerivation rec {
+  pname = "boxfort";
+  version = "38fe630";
+
+  src = fetchFromGitHub
+    { owner = "Snaipe";
+      repo  = "BoxFort";
+      rev   = "38fe63046fbabcae34ebc2ee9867d990ac28c4c5";
+      sha256 = "03zy982d1frpk9fr4hbp1ql1ah06s0v6dy7a56d1zl3jjjljhrhn";
+    };
+
+  buildInputs =
```

Comment:
`meson`, `ninja`, `cmake`, `pkg-config`, and potentially `git` should be in `nativeBuildInputs` instead of `buildInputs`.

Reply:
Fixed